org.sebicodes.Prt.yml

```diff
@@ -0,0 +1,19 @@
+app-id: org.sebicodes.Prt
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: runner.sh
+finish-args:
+  - --share=network
+modules:
+  - name: app
```

Comment:
it's usualla a practice to use a module name that relate to the application. not something so generic.

Reply:
FIxed. Is it right now?